views/topologicalSort.pug

```diff
@@ -1,9 +1,263 @@
-extends layout
+doctype html
+html
+  head
+    meta(charset='UTF-8')
+    title Topological Sorting
+  style.
+    .axis path,
```

Comment:
This part is the same as the css file as Boyi added, you can reuse it (after rename).

Reply:
Thank you for your reminding.